### DISCUSSION TITLE
Şehirler arası yol seçeneği sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#10] [Zeka Sorusu]
Bir ülkede ,,A B C ve D şehirleri vardır. A şehriyle ,,B C D şehirleri arasındaki direkt yolların sayısı sırasıyla 2, 2, 3 tür. B şehriyle ,CD şehirleri arasındaki direkt yolların sayısı sırasıyla 5,1 ; C ile D arasındaki direkt yolların sayısı ise 3 ’tür. Buna göre A şehrinden D şehrine, bir şehirden en fazla bir kez geçerek kaç farklı şekilde gidilebilir?

**Comment by FatihOnder81** [comment]
41 buldum dediğiniz bilgilere göre tablo çıkardım hocam

**Reply by sonsuzus** [reply]
Nasıl çözdüğünü anlatabilirsin, resim de ekleyebilirsin. Ayrıca etiketleri arasında program olanları programlama yoluyla bulmanızı istiyorum. Kod bekliyorum yani soruyu çözen.
